File: src/test/unit/controller/MainTest.java
package controller;

import org.junit.jupiter.api.Assertions;

public class MainTest {

    void getMain_shouldBe_MyNameIsGivenName() {
        String name = "Karmo";
        String given = Main.Companion.main(name);
        Assertions.assertEquals("My name is: $name", given);

    }
}

File: src/test/unit/controller/MainTest.kt
package unit.controller

import controller.Main.Companion.main
import org.junit.Test
import kotlin.test.assertEquals

internal class MainTest {
    @Test
    fun testMain() {
        val name = "Karmo"
        val given = main(name)
        assertEquals("Name is: $name", given)
    }
}